### PR TITLE
Create Resource page

### PR DESCRIPTION
**Pull request by AliciadW**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -89,6 +90,8 @@ class Header extends Component {
       title = titleOfStory
```
Comment (schalkventer) [inline src/components/Header.js]: Perhaps turn this into a funtion that uses a switch (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) with returns.

File: src/views/ResourcePage.jsx [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+import Footer from '../components/Footer';
 
 const Markup = (props) => (
   <Fragment>
-    <h1>Resource Page</h1>
-    <Preview>
-      {JSON.stringify(props, 0, 2)}
-    </Preview>
+
+    <Helmet>
```
Comment (schalkventer) [inline src/views/ResourcePage.jsx]: Also can you look at removing these and rather using SVG icons and using https://www.npmjs.com/package/gatsby-plugin-web-font-loader?

File: src/components/Header.js (hunk 1 above)
Comment (AliciadW) [reply src/components/Header.js]: I'll do that.

File: src/views/ResourcePage.jsx (hunk 2 above)
Comment (AliciadW) [reply src/views/ResourcePage.jsx]: Sure, will do.

**Pull request merged**